test/fakeintake/api/api.go

```diff
@@ -0,0 +1,10 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package api
+
+type GetPayloadResponse struct {
```

Comment:
The name feels like a function name. We're mostly interested in what the type represents more than where it's used.
I think something like `RawPayloads` or `Payloads`

Reply:
I wanted to pass the info that this is the model of a `GET` response. Maybe `APIFakeIntakePayloadsGETResponse` or `APIPayloadsGETResponse` ? It would follow the naming pattern used in frontend code